packs/travis_ci/actions/get_git_token.py

```diff
@@ -0,0 +1,22 @@
+from lib.action import TravisCI
+import yaml
+import requests
+
+
+class GetGitToken(TravisCI):
+    def run(self):
+        username = self.config['username']
+        password = self.config['password']
+        url = 'https://api.github.com/authorizations'
+        host = 'api.github.com'
+        ContentType = 'application/json'
+        data = '{"scopes": ["read:org", "user:email",\
+                "repo_deployment","repo:status", "write:repo_hook"],\
+                "note": "toke for travis api"}'
+        headers = {}
+        headers['Host'] = host
+        headers['Content-Type'] = ContentType
+        response = requests.post(url, data=data, auth=(username, password),
+                                 headers=headers)
+        res = yaml.load(response.content)
```

Comment:
There is no need to use yaml, API returns JSON which is already parsed by requests and available in `response.json()`.

See https://github.com/StackStorm/st2contrib/commit/82b7df9589409c47e63f77a846f67e7f38519e25 for details.